app/Exports/WordsExport.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Exports;
+
+use Maatwebsite\Excel\Events\BeforeExport;
+use Maatwebsite\Excel\Events\BeforeWriting;
+use Maatwebsite\Excel\Files\LocalTemporaryFile;
+use Maatwebsite\Excel\Excel;
+use Maatwebsite\Excel\Concerns\Exportable;
+use Maatwebsite\Excel\Concerns\WithEvents;
+use PhpOffice\PhpSpreadsheet\Cell\DataValidation;
+
+class WordsExport implements  WithEvents
+{
+    use Exportable;
+
+    public function registerEvents(): array 
+    {
+        $types = [1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
mảng gì đây em?

Reply:
mảng các option trong file excel ạ

Reply:
em đưa vào config và comment not lại ý nghĩa của mảng nhé, như này khó hiểu quá

Reply:
sửa chỗ này đi em